pkg/bindings/images/build.go

```diff
@@ -604,8 +605,8 @@ func Build(ctx context.Context, containerFiles []string, options entities.BuildO
 	var id string
 	for {
 		var s struct {
-			Stream string `json:"stream,omitempty"`
-			Error  string `json:"error,omitempty"`
+			Stream string                 `json:"stream,omitempty"`
+			Error  *jsonmessage.JSONError `json:"errorDetail,omitempty"`
 		}
```

Comment:
This could re-use the new struct, could it?

Reply:
The deprecated warning is still being used in the docker compat bindings, so I can't remove.

Reply:
Can we use another/new struct?  I would have hard time trying to understand the reasoning in the future.  A new struct wouldn't raise the question of why only parts are used and what's deprecated etc.